src/model/constants/DraftHandleValue.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule DraftHandleValue
+ * @flow
+ */
+
+/*eslint-disable no-bitwise*/
```

Comment:
Did lint complain about the bitwise below? I think we should avoid doing that for exported Flow types. cc @zpao